hawkular-inventory-parent/hawkular-inventory-service/pom.xml

```diff
@@ -236,6 +236,30 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <!-- FIXME: this is for arquillian only, there's probably another way to do it -->
```

Comment:
@lucasponce / @jshaughn , I'm copying test resources in the WF deployment for integration tests, I don't know if there's another way to do that.
Basically what I want is to have my "test.yml" file under "$wf/standalone/configuration/hawkular" for tests only. But I don't want it to appear in the dist.

Reply:
Yes, you need to add it in the @Deployment method of the test as a resource